# Billionaires Dataset - EDA

In [1]:
import pandas as pd

In [2]:

filename = 'Billionaire.csv'
complete_filepath =filename 
data = pd.read_csv(complete_filepath)

### Part 1 Data Exploration

In [4]:
data.head()

Name NetWorth        Country         Source  Rank
0                Jeff Bezos   $177 B  United States         Amazon     1
1                 Elon Musk   $151 B  United States  Tesla, SpaceX     2
2  Bernard Arnault & family   $150 B         France           LVMH     3
3                Bill Gates   $124 B  United States      Microsoft     4
4           Mark Zuckerberg    $97 B  United States       Facebook     5

In [5]:
data.shape

(2755, 5)

In [6]:
data.columns

Index(['Name', 'NetWorth', 'Country', 'Source', 'Rank'], dtype='object')

#### Questions to Ask About Data

So we have names, the networth, the country of origin, source of income and ranking for all these billionaires. When you have any datasets, there are some fundamental questions you would want to ask yourself and then find those answers within the data. For instance, for our dataset we can ask ourselves:
1. How many billionaires are there in the list?
2. Who are the top 10 billionaire?
3. Who are the top billionaire by country of origin?
4. What country has the highest number of billionaires?
5. What country has the least number of billionaires?
6. What is the average and median networth of each billionaire?
7. Are there any outliers i.e. billionaires that have exceptionally high networth compared to others?
8. For each country, what is the cumulative networth of its billionaires?
9. What is the maximum and minimum networth of billionaires overall and by country?
10. Are there are any billionaires with the same source of income? 

#### But First, Data Cleaning 

Before we dive into answering these questions, we need to make sure of the following (and this applies to all datasets that you will encounter in future so keep this checklist in mind):
1. are there any null or missing values in the data?
2. are the data types of each column what you are expecting it to be?

Let's tackle these questions one by one. 



#### Handling Missing Values

In [7]:
data.isna()

Name  NetWorth  Country  Source   Rank
0     False     False    False   False  False
1     False     False    False   False  False
2     False     False    False   False  False
3     False     False    False   False  False
4     False     False    False   False  False
...     ...       ...      ...     ...    ...
2750  False     False    False   False  False
2751  False     False    False   False  False
2752  False     False    False   False  False
2753  False     False    False   False  False
2754  False     False    False   False  False

[2755 rows x 5 columns]

In [8]:
data.isna().any()

Name        False
NetWorth    False
Country     False
Source      False
Rank        False
dtype: bool

Great. So there are no missing values in our dataset. What if there had been how would we handle that?
This requires a bit of thought process on your part. since we have individual billionaires, and if there were missing values say in country of origin, you would have to manually search and fill those spaces since no mathematical option (such as using averages) would make sense. Or you could ideally just drop those values specially if their ranks are in the lower half of the dataset for instance. 

In [10]:
# since our data does not have any nan values, using dropna() makes no difference.
# we see no difference is made since the data is still the same shape
dropped_data = data.dropna()
print(dropped_data.shape)

(2755, 5)


In [11]:
data.equals(dropped_data)

True

#### Handling Data Types

We need to ensure that data has the right type to perform calculations on. There are two columns in our dataset that are numeric i.e. `Rank` and `Networth`. 

In [18]:
data.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2755 entries, 0 to 2754
Data columns (total 5 columns):
 #   Column    Non-Null Count  Dtype 
---  ------    --------------  ----- 
 0   Name      2755 non-null   object
 1   NetWorth  2755 non-null   object
 2   Country   2755 non-null   object
 3   Source    2755 non-null   object
 4   Rank      2755 non-null   int64 
dtypes: int64(1), object(4)
memory usage: 107.7+ KB


We see that Rank has the right data type i.e. int64 (which is the same as integer). However, NetWorth has a datatype of object, while it should be float or integer. Let's focus on this column first and fix its data type.

In [19]:
# look at data for Networth column 
data['NetWorth']

0       $177 B
1       $151 B
2       $150 B
3       $124 B
4        $97 B
         ...  
2750      $1 B
2751      $1 B
2752      $1 B
2753      $1 B
2754      $1 B
Name: NetWorth, Length: 2755, dtype: object

In [20]:
# let's look at the data type of one of the values.
first_value = data['NetWorth'].iloc[0]

print(type(first_value))

<class 'str'>


We see that it has $ and B to represent column values as currency. However, they also prevent it from being interpreted as numeric values. We can fix that by creating a new column that only contains the numeric content of this column. Currently, object data type is basically string so we need to cut down the string to only take the numeric value out of it.

In [21]:
# let's test our logic by subsetting the first value to acquire the numeric content. 
# Strings can be subset like arrays so here, if we use [0] we can access the first item of this string which is a $
print(first_value[0])
print(first_value[1:])

$
177 B


In [22]:
# similarly, the last two values are a space and a B
first_value[-2:]

' B'

In [23]:
# so combining both these indexing, we get the numeric content. However, that would still be string type
subset = first_value[1:-2]
print(subset)
print(type(subset))

177
<class 'str'>


In [24]:
# so we need one more step to convert this data type into numeric. 
converted = float(subset)
type(converted)

float

In [15]:
# defining a user-defined function called fix_networth(value)
# we already have the logic figured out, let's just copy the code here.
def fix_networth(value):
    subset = value[1:-2]
    converted = float(subset)
    return converted 


# this function takes one value at a time and fixes it to the format and data type we want. 
new_column = data['NetWorth'].apply(fix_networth)
print(new_column)

0       177.0
1       151.0
2       150.0
3       124.0
4        97.0
        ...  
2750      1.0
2751      1.0
2752      1.0
2753      1.0
2754      1.0
Name: NetWorth, Length: 2755, dtype: float64


In [16]:
#Let's store it in a new column called ['NetWorth ($B)']
data['NetWorth ($B)'] = new_column
print(data.info())

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2755 entries, 0 to 2754
Data columns (total 6 columns):
 #   Column         Non-Null Count  Dtype  
---  ------         --------------  -----  
 0   Name           2755 non-null   object 
 1   NetWorth       2755 non-null   object 
 2   Country        2755 non-null   object 
 3   Source         2755 non-null   object 
 4   Rank           2755 non-null   int64  
 5   NetWorth ($B)  2755 non-null   float64
dtypes: float64(1), int64(1), object(4)
memory usage: 129.3+ KB
None


That's great! Now we are almost ready to get started to answering our questions. However, let's get rid of the old NetWorth column since its of no use to us anymore.

In [17]:
# drop old networth column
data = data.drop(columns=['NetWorth'])

In [28]:
data.head()

Name        Country         Source  Rank  NetWorth ($B)
0                Jeff Bezos  United States         Amazon     1          177.0
1                 Elon Musk  United States  Tesla, SpaceX     2          151.0
2  Bernard Arnault & family         France           LVMH     3          150.0
3                Bill Gates  United States      Microsoft     4          124.0
4           Mark Zuckerberg  United States       Facebook     5           97.0

Looking great! Let's dig in!

### Answering Data Questions
Re-stating our questions here for ease.
1. How many billionaires are there in the list?
2. Who are the top 10 billionaire?
3. Who are the top billionaire by country of origin?
4. What country has the highest number of billionaires?
5. What country has the least number of billionaires?
6. What is the average and median networth of each billionaire?
7. Are there any outliers i.e. billionaires that have exceptionally high networth compared to others?
8. For each country, what is the cumulative networth of its billionaires?
9. What is the maximum and minimum networth of billionaires overall and by country?
10. Are there are any billionaires with the same source of income? 

In [12]:
# Our first question, how many Billionaires in our dataset?

data['Name'].unique()

array(['Jeff Bezos', 'Elon Musk', 'Bernard Arnault & family', ...,
       'Zhao Meiguang', 'Zhong Naixiong', 'Zhou Wei family'], dtype=object)

In [19]:
# this returns an array. 
unique_values = data['Name'].unique()
unique_values.shape

(2752,)

Looks like we had some duplicate values there. Back to data cleaning board.

#### Data Cleaning, Again

Now we have some duplicated values. We can use the duplicated() function to find out if there are any duplicates.

In [20]:
data['Name'].duplicated()

0       False
1       False
2       False
3       False
4       False
        ...  
2750    False
2751    False
2752    False
2753    False
2754    False
Name: Name, Length: 2755, dtype: bool

That's a lot of Falses, but somewhere in there is a true. Now what we will do is use a new way to subset dataframe. We will use conditional **boolean masks** to retreive only those values that are marked duplicated by pandas. For that, we will use the following syntax

`df[conditional expression]`

That translates to:

`data[data['Name'].duplicated()]`

What this means is, return to us the original dataset where there are marked duplicates as we computed on the `Name` column.
the `conditional expression` can be applied on any column within the dataframe. We will see more examples of this later. For now, let's see our duplicated values.

In [21]:
data[data['Name'].duplicated()]

Name Country                  Source  Rank  \
1625          Robert Miller  Canada  electronics components  1580   
1919  Wang Yanqing & family   China   carbon fiber products  1833   
1976                  Li Li   China         pharmaceuticals  1931   

      NetWorth ($B)  
1625            2.0  
1919            1.7  
1976            1.6

Looks like we have these three duplicated values with us. We can use the same conditional expression only this time we know the names we are looking for. So, now the expression looks something like `data['Name'] == 'Robert Miller'` for instance, where we want to retrieve all rows where the name is Robert Miller. We know it appears at least more than once since it has been marked duplicate. Let's have a look at all our duplicated names. 

In [22]:
data[data['Name'] == 'Robert Miller']

Name         Country                  Source  Rank  \
1331  Robert Miller  United Kingdom                  retail  1299   
1625  Robert Miller          Canada  electronics components  1580   

      NetWorth ($B)  
1331            2.4  
1625            2.0

In [23]:
data[data['Name'] == 'Wang Yanqing & family']

Name Country                 Source  Rank  \
700   Wang Yanqing & family   China   electrical equipment   680   
1919  Wang Yanqing & family   China  carbon fiber products  1833   

      NetWorth ($B)  
700             4.2  
1919            1.7

In [24]:
data[data['Name'] == 'Li Li']

Name Country           Source  Rank  NetWorth ($B)
693   Li Li   China       healthcare   680            4.2
1976  Li Li   China  pharmaceuticals  1931            1.6

Now there are two different scenarios here.
1. In case of Robert Miller, they are not the same. We notice this because their country, source and networth also differs. Which means they are two different people.
2. In case of the other two entries, both have the same country and name however, different source and networths.


For 1, we do a quick google search and find out that these are indeed two different people. We have an Canadian Robert Miller[https://en.wikipedia.org/wiki/Robert_Miller_(businessman)] and a American Robert Miller[https://en.wikipedia.org/wiki/Robert_Warren_Miller]

For 2, we also do some search and find Forbes mention their rank at 1976[https://www.forbes.com/profile/li-li-2/?sh=7caff2f216d9] for Li Li and at 680[https://www.forbes.com/profile/wang-yanqing/?sh=53096d9240a8] for Wang Yanqing & Family. 


So our approach to handling duplicates here will be:
1. In case of Robert Miller, we will let this be. 
2. In case of Wang Yanqing & Family, we will drop carbon fiber products entry.
3. In case of Li Li, we will drop healthcare entry. 

In [25]:
# Since we now know that Name is not a unique list (we have two Robert Millers) so we have another problem to deal with.
# We will use drop() with argument for index where we will pass the index value of the rows we are about to drop.
fixed_data = data.drop(index=[693, 1919])

Now we know there is one duplicate name in Name column hence we can no longer use that for total count of billionaire.
We will however be able to use .shape since we have catered to the duplicates issue and now the total length correctly reflects the total number of billionaires.

In [26]:
# total billionaires
print(fixed_data.shape)

(2753, 5)


# Part 2

We will continue with our previous dataset and attempt to answer our questions.

### Answering Data Questions
Re-stating our questions here for ease.
1. How many billionaires are there in the list?
2. Who are the top 10 billionaire?
3. Who are the top billionaire by country of origin?
4. What country has the highest number of billionaires?
5. What country has the least number of billionaires?
6. What is the average and median networth of each billionaire?
7. Are there any outliers i.e. billionaires that have exceptionally high networth compared to others?
8. For each country, what is the cumulative networth of its billionaires?
9. What is the maximum and minimum networth of billionaires overall and by country?
10. Are there are any billionaires with the same source of income? 

Let's have a look at our data set that we have so far cleaned stored in `fixed_data` variable.

In [27]:
fixed_data.head()

Name        Country         Source  Rank  NetWorth ($B)
0                Jeff Bezos  United States         Amazon     1          177.0
1                 Elon Musk  United States  Tesla, SpaceX     2          151.0
2  Bernard Arnault & family         France           LVMH     3          150.0
3                Bill Gates  United States      Microsoft     4          124.0
4           Mark Zuckerberg  United States       Facebook     5           97.0

Q1. How many billionaires are there in the list?

Answer: Since we have removed our duplicates and can use .shape to determine the number of rows hence our total billionaires. 

In [28]:
print('Number of Billionaires in our list: ', fixed_data.shape[0])

Number of Billionaires in our list:  2753


Q2. Who are the top 10 billionaires?

Answer: Now there are various ways to do this. You can use the `Rank` column and get values that `<= 10`. You could sort the values by `NetWorth` and then either use `head(n)` or index the first ten rows.

In [29]:
# let's use the rank column approach. 
top_ten = fixed_data[fixed_data['Rank'] <= 10]
top_ten['Name']

0                  Jeff Bezos
1                   Elon Musk
2    Bernard Arnault & family
3                  Bill Gates
4             Mark Zuckerberg
5              Warren Buffett
6               Larry Ellison
7                  Larry Page
8                 Sergey Brin
9               Mukesh Ambani
Name: Name, dtype: object

#### nlargest() and nsmallest() functions
Another easier way to retrieve top n (or bottom n) values based on a column is to use the `nlargest()` (or the `nsmallest()`) functions on the data frame.

In [30]:
# alternate approach uses nlargest which retrieves all data for the top N values based on the specified column. 

alt_top_ten = fixed_data.nlargest(10, columns='NetWorth ($B)')
alt_top_ten['Name']

0                  Jeff Bezos
1                   Elon Musk
2    Bernard Arnault & family
3                  Bill Gates
4             Mark Zuckerberg
5              Warren Buffett
6               Larry Ellison
7                  Larry Page
8                 Sergey Brin
9               Mukesh Ambani
Name: Name, dtype: object

Try the other approaches on your own and see if you get the same results.

Q3. Who are the top billionaire by country of origin?

Answer: For this we will have to use the groupby function to group our data on set

In [40]:
# use group by to find the max value for each group
country=fixed_data.groupby('Country')['NetWorth ($B)']
indexes_we_want=country.idxmax().values
fixed_data.iloc[indexes_we_want,:]

Name         Country       Source  Rank  NetWorth ($B)
597   Issad Rebrab & family         Algeria         food   589            4.8
441         Marcos Galperin       Argentina   e-commerce   440            6.1
69            Gina Rinehart       Australia       mining    70           23.6
55      Dietrich Mateschitz         Austria     Red Bull    56           26.9
228           Eric Wittouck         Belgium  investments   228           10.0
...                     ...             ...          ...   ...            ...
112         James Ratcliffe  United Kingdom    chemicals   113           17.0
0                Jeff Bezos   United States       Amazon     1          177.0
965            Carlo Fidani          Canada  real estate   956            3.2
349         Pham Nhat Vuong         Vietnam  diversified   344            7.3
2089        Lorinc Meszaros         Hungary  diversified  2035            1.5

[70 rows x 5 columns]

Q4 and Q5. Which countries have the highest and the lowest number of billionaires?

Answer: We will group data by country and then use `.count()` function on the `Name` column.

In [87]:
# group data
grouped_by_country = fixed_data.groupby('Country')
# use .count() function on Name column
country_count = grouped_by_country['Name'].count()

# now we need to access the highest and the lowest values. Let's use nlargest and nsmallest()
print('Country with highest number of billionaires: \n', country_count.nlargest(1))
print('\n')
print('Country with lowest number of billionaires: \n', country_count.nsmallest(1))

Country with highest number of billionaires: 
 Country
United States    724
Name: Name, dtype: int64


Country with lowest number of billionaires: 
 Country
Algeria    1
Name: Name, dtype: int64


In [91]:
print('Highest Billionaires are in', country_count.idxmax(), 'which are', country_count.max())

print('Least Billionaires are in', country_count.idxmin(), 'which are', country_count.min())

Highest Billionaires are in United States which are 724
Least Billionaires are in Algeria which are 1


Q6. What is the average and median networth of each country's billionaires?

Answer: We will again group data by countries and then use mean and median on NetWorth columns

In [99]:
# group data by country
grouped_by_country = fixed_data.groupby('Country')

# get average networth 
mean_networth = grouped_by_country['NetWorth ($B)'].mean()

mean_networth.head(10)

Country
Algeria      4.800000
Argentina    3.060000
Australia    4.125000
Austria      5.225000
Belgium      4.200000
Brazil       3.256923
Canada       3.610937
Chile        4.744444
China        4.048077
Colombia     5.240000
Name: NetWorth ($B), dtype: float64

In [98]:
# get median networth 
median_networth = grouped_by_country['NetWorth ($B)'].median()

median_networth.head(10)

Country
Algeria      4.80
Argentina    2.40
Australia    2.30
Austria      2.90
Belgium      1.50
Brazil       2.20
Canada       2.15
Chile        2.40
China        2.00
Colombia     3.90
Name: NetWorth ($B), dtype: float64

#### Food for Thought

You will notice that for some countries median and mean are the same while for others they may be different. Why do you think is that?

Q7 and Q8: Are there any outliers i.e. billionaires that have exceptionally high networth compared to others? For each country, what is the cumulative networth of its billionaires?

Answer: We can answer these questions together. First let's tackle Q8 i.e. cumulative networths by each country. This is relatively simple. AS we computed median and mean, we simply need to use the `sum()` function instead after grouping by `Country`.

In [103]:
# group data by country
grouped_by_country = fixed_data.groupby('Country')

# generate total wealth by country
cum_wealth = grouped_by_country['NetWorth ($B)'].sum()


cum_wealth.nlargest(10)

Country
United States     4398.4
China             2526.0
Germany            625.5
India              596.4
Russia             586.0
France             512.2
Hong Kong          448.2
Canada             231.1
Japan              225.4
United Kingdom     213.9
Name: NetWorth ($B), dtype: float64

Now back to Q7. Outliers are values that are generally far off from the general dataset, something that skews the mean. 
For instance, in an undergrad first year class, you would normally expect students to be of age 17-19. However, if there is one new student aged 35 there, it would be regarded as an outlier. 
We will learn more about these measures in a later section however for now this definition should suffice. 

One way to detect outliers is by using quantiles. Quantiles is a way of dividing data into equally cut portions based on the data such that highest quantile contains highest values of the data and vice versa for lowest. The values of quantiles are represented in percentages between 0% to 100% (or between 0 to 1). Normally, if a dataset contains outliers, you can compute the mean and median and if the values are significantly different, then you can compute the quantile to see what the highest values are. You could also simply use nlargest() to find the largest values of the dataset but that requires a bit of trial and error to find the right value of n since the number of outliers is unknown.


For us, highest values are generally outliers so we will use 0.99 as our value for quantile which means show us the breakpoint where top 1% of the highest value are. Similarly, we can also use 0.95 for breakpoint for top 5% of the data and so on.

In [108]:
# we need to find the quantile for entire dataset
top_1_pct = fixed_data['NetWorth ($B)'].quantile(0.99)

# subset data to retrieve networth greater than these values
fixed_data[fixed_data['NetWorth ($B)'] > top_1_pct]

Name        Country  \
0                              Jeff Bezos  United States   
1                               Elon Musk  United States   
2                Bernard Arnault & family         France   
3                              Bill Gates  United States   
4                         Mark Zuckerberg  United States   
5                          Warren Buffett  United States   
6                           Larry Ellison  United States   
7                              Larry Page  United States   
8                             Sergey Brin  United States   
9                           Mukesh Ambani          India   
10                         Amancio Ortega          Spain   
11  Francoise Bettencourt Meyers & family         France   
12                         Zhong Shanshan          China   
13                          Steve Ballmer  United States   
14                             Ma Huateng          China   
15              Carlos Slim Helu & family         Mexico   
16                           Alice Walton  United States   
17                             Jim Walton  United States   
18                             Rob Walton  United States   
19                      Michael Bloomberg  United States   
20                      Colin Zheng Huang          China   
21                        MacKenzie Scott  United States   
22                         Daniel Gilbert  United States   
23                  Gautam Adani & family          India   
24                   Phil Knight & family  United States   
25                                Jack Ma          China   
26                           Charles Koch  United States   
27                    Julia Koch & family  United States   

                         Source  Rank  NetWorth ($B)  
0                        Amazon     1          177.0  
1                 Tesla, SpaceX     2          151.0  
2                          LVMH     3          150.0  
3                     Microsoft     4          124.0  
4                      Facebook     5           97.0  
5            Berkshire Hathaway     6           96.0  
6                      software     7           93.0  
7                        Google     8           91.5  
8                        Google     9           89.0  
9                   diversified    10           84.5  
10                         Zara    11           77.0  
11                      L'Oréal    12           73.6  
12   beverages, pharmaceuticals    13           68.9  
13                    Microsoft    14           68.7  
14               internet media    15           65.8  
15                      telecom    16           62.8  
16                      Walmart    17           61.8  
17                      Walmart    18           60.2  
18                      Walmart    19           59.5  
19                 Bloomberg LP    20           59.0  
20                   e-commerce    21           55.3  
21                       Amazon    22           53.0  
22                Quicken Loans    23           51.9  
23  infrastructure, commodities    24           50.5  
24                         Nike    25           49.9  
25                   e-commerce    26           48.4  
26              Koch Industries    27           46.4  
27              Koch Industries    27           46.4

We have a total of 2753 values and 1% of the data is 27 values. That is what we get. Notice how the top 4 values are significantly larger compared to the rest of the values for Net Worth. Now that we know that the top 4 are the outlier-ish values, we can use the `nlargest()` to obtain the top 4.

In [110]:
# get the outlier values
print("Outliers in the dataset are as follows")
fixed_data.nlargest(4, columns='NetWorth ($B)')

Outliers in the dataset are as follows


Name        Country         Source  Rank  NetWorth ($B)
0                Jeff Bezos  United States         Amazon     1          177.0
1                 Elon Musk  United States  Tesla, SpaceX     2          151.0
2  Bernard Arnault & family         France           LVMH     3          150.0
3                Bill Gates  United States      Microsoft     4          124.0

Q9.What is the maximum and minimum networth of billionaires overall and by country?

Answer: For overall, we can simply call the `.max()` and `.min()` functions on our dataset. For country wise, we can do the same after grouping data by country.

Are there are any billionaires with the same source of income?

In [113]:
# find max and min overall
print('Highest NetWorth of a Billionaire is', fixed_data['NetWorth ($B)'].max(), '$B')
print('Lowest NetWorth of a Billionaire is', fixed_data['NetWorth ($B)'].min(), '$B')

Highest NetWorth of a Billionaire is 177.0 $B
Lowest NetWorth of a Billionaire is 1.0 $B


In [117]:
# find max and min by country
# group data 
grouped_by_country = fixed_data.groupby('Country')
# get max billionaire wealth per country and sort countries
grouped_by_country['NetWorth ($B)'].max().sort_index()

Country
Algeria             4.8
Argentina           6.1
Australia          23.6
Austria            26.9
Belgium            10.0
                  ...  
United Kingdom     17.0
United States     177.0
Venezuela           3.2
Vietnam             7.3
Zimbabwe            1.5
Name: NetWorth ($B), Length: 70, dtype: float64

In [118]:
# get max billionaire wealth per country and sort countries 
grouped_by_country['NetWorth ($B)'].min().sort_index()

Country
Algeria           4.8
Argentina         1.3
Australia         1.0
Austria           1.1
Belgium           1.1
                 ... 
United Kingdom    1.0
United States     1.0
Venezuela         3.2
Vietnam           1.2
Zimbabwe          1.5
Name: NetWorth ($B), Length: 70, dtype: float64

Q10. Are there are any billionaires with the same source of income?

Answer: For this purpose, we can group data by source of income and then count billionaires in each section. After that sort by highest to lowest to see billionaires that have the same source of income and what exactly is that source.

In [122]:
# group by source of income
group_by_source = fixed_data.groupby('Source')
# count the billionaires using name 
source_count = group_by_source['Name'].count()
# sort by highest to lowest count 
source_count.sort_values(ascending=False)

Source
real estate                  171
pharmaceuticals               99
investments                   90
diversified                   85
software                      62
                            ... 
hotels, energy                 1
hotels, diversified            1
supermarkets, investments      1
hospitals, health care         1
3D printing                    1
Name: Name, Length: 923, dtype: int64

Based on our calculations, the top 3 sources for becoming billionaires appear to be:
1. Real Estate 
2. Pharmaceuticals 
3. Investments 


Do you belong to any of these fields?